frontend/src/scenes/data-pipelines/batch-exports/BatchExportEditForm.tsx

```diff
@@ -455,14 +455,173 @@ export function BatchExportsEditFields({
+                                    <LemonField
+                                        name="redshift_s3_bucket_aws_access_key_id"
+                                        label="AWS Access Key ID"
+                                        className="flex-1"
+                                    >
+                                        <LemonInput
+                                            placeholder={isNew ? 'e.g. AKIAIOSFODNN7EXAMPLE' : 'Leave unchanged'}
+                                        />
+                                    </LemonField>
+
+                                    <LemonField
+                                        name="redshift_s3_bucket_aws_secret_access_key"
+                                        label="AWS Secret Access Key"
+                                        className="flex-1"
+                                    >
+                                        <LemonInput
+                                            placeholder={isNew ? 'e.g. secret-key' : 'Leave unchanged'}
+                                            type="password"
+                                        />
+                                    </LemonField>
+                                </div>
+
+                                <LemonField
+                                    name="authorization_mode"
+                                    label="Authorization"
+                                    className="flex-1"
+                                    info={
+                                        <>
+                                            Redshift needs to authenticate to COPY data from your S3 bucket. Choose
+                                            whether to assume an IAM role or to provide it with credentials to access
+                                            the S3 bucket.
+                                        </>
+                                    }
+                                >
+                                    <LemonSelect
+                                        options={[
+                                            { value: 'IAMRole', label: 'IAM Role' },
+                                            { value: 'Credentials', label: 'Credentials' },
+                                        ]}
+                                    />
+                                </LemonField>
+
+                                {batchExportConfigForm.authorization_mode === 'IAMRole' && (
+                                    <>
+                                        <LemonField name="redshift_iam_role" label="IAM Role ARN" className="flex-1">
+                                            <LemonInput placeholder="e.g. arn:aws:iam::<aws-account-id>:role/<role-name>" />
+                                        </LemonField>
+                                    </>
+                                )}
+
+                                {batchExportConfigForm.authorization_mode === 'Credentials' && (
```

Comment:
I'm not sure how easy it is, but I it would be nice to have a "Use same credentials" checkbox or similar to copy over the credentials from above, since I imagine a lot of users would use the same ones.